lib/rules/no-const-outside-module-scope.js

```diff
@@ -1,7 +1,12 @@
 'use strict';
 
+/** @type {import('eslint').Rule.RuleModule} */
```

Comment:
Explanation of this jsdoc comment: https://github.com/ember-cli/eslint-plugin-ember/pull/1301